### ISSUE TITLE
AdaptiveETA gives negative ETA

### ISSUE THREAD

**Issue by krist10an** [WoLpH/python-progressbar#56]
Thanks for a great package!

I've noticed that when using the AdaptiveETA widget and my download speed is not uniform you will get a negative ETA. It would be better if it just said 00:00.

Script for reproducing:

```
from progressbar import ProgressBar, AdaptiveETA
import time

pb = ProgressBar(widgets=[AdaptiveETA()], max_value=20)

for i in range(20):
    pb.update(i)
    time.sleep(0.1)
    if i < 10:
        time.sleep(0.5)
print("")
```
Output is:

```
ETA:  -1 day, 23:59:56
```

**Comment by WoLpH** [comment]
That's indeed a bit odd, guess there should be a `max(0, delta)` somewhere in there :)

I'll take a look at it soon

**Comment by krist10an** [comment]
This seems to have been fixed in e48fc913f89028560033a5ed6bfaa010a5e80d2e :+1:  Thanks!

**Comment by WoLpH** [comment]
That's great to hear. Sorry for not responding sooner I've just been really busy
